raster/r.geomorphon/r.geomorphon.html

```diff
@@ -149,15 +149,15 @@ <h3>Geomorphon calculation: extraction of terrestrial landforms</h3>
 # verify terrestrial landforms found in DEM
 r.category eu_dem_25m_geomorph
  1  flat
- 2  summit
+ 2  peak
```

Comment:
To avoid confusion, should the second manual page example also be updated? Or defined, that "summit = peak" to keep it understandable?